rjc_purchase_order_form/reports/purchase_order_layout.xml

```diff
@@ -0,0 +1,299 @@
+                            </th>
+                            <th class="text-right">
+                                <br>
+                                    <strong>หน่วยละ</strong>
+                                </br>
+                                <br>
+                                    <strong>Unit Price</strong>
+                                </br>
+                            </th>
+                            <th class="text-right">
+                                <br>
+                                    <strong>จำนวนเงิน</strong>
+                                </br>
+                                <br>
+                                    <strong>Amount</strong>
+                                </br>
+                            </th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <t t-set="i" t-value="1"/>
+                        <tr t-foreach="o.order_line" t-as="line">
+                            <td>
+                                <span t-esc="i"/>
+                                <t t-set="i" t-value="i+1"/>
+                            </td>
+                            <td>
+                                <span t-field="line.name"/>
+                            </td>
+                            <td class="text-right">
+                                <span t-field="line.product_qty"/>
+                            </td>
+                            <td class="text-right">
+                                <span t-field="line.product_uom.name" groups="uom.group_uom"/>
+                            </td>
+                            <td class="text-right">
+                                <span t-field="line.price_unit"/>
+                            </td>
+                            <td class="text-right">
+                                <span t-field="line.price_subtotal" t-options='{"widget": "monetary", "display_currency": o.currency_id}'/>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div class="row">
+                <div class="col-6">
+                    <p t-field="o.notes"/>
+                    <div class="oe_structure"/>
+                    (<span t-esc="o.amount_text(o.amount_total)"/>)
+                </div>
+                <div class="col-6" style="margin-left:20px">
+                    <table class="table table-sm" border="1" style="width:100%;">
+                        <tr class="border-black">
+                            <td>
+                                <strong>ราคาก่อนหักส่วนลด</strong>
+                            </td>
+                            <td>
+                                <strong>Subtotal</strong>
+                            </td>
+                            <td class="text-right">
+                                <span t-esc="'{0:,.2f}'.format(o.amount_before_discount())" />฿
```

Comment:
```suggestion
                                <span t-esc="'{0:,.2f}'.format(amount_before_discount)"
                                      t-options='{"widget": "monetary", "display_currency": o.currency_id}'/>
```
t-options is function for show currency. I think it's work.
you shouldn't hardcode. If currency is not thai?